src/Config.php

```diff
@@ -29,21 +29,23 @@ class Config
     private $gcp_call_invoker;
 
     /**
-     * @param string  $hostname Which API we want to manage the connection.
+     * @param string  $target The target API we want to manage the connection.
      * @param \Grpc\Gcp\ApiConfig   $conf
      * @param CacheItemPoolInterface $cacheItemPool A pool for storing configuration and channels
```

Comment:
I'm not sure the standard in the library, but if you're open to it could we please document the `RuntimeException`?

```
@throws \RuntimeException When a failure occurs while attempting to attach to shared memory.
```

Reply:
done.